docs/all-resources-must-have-delete.md

```diff
@@ -18,15 +18,15 @@ The resource {resourceName} does not have a corresponding delete operation.
 
 ## Description
 
-All top level proxy and (tracked at all levels) resources MUST support delete.
+All proxy and tracked resources MUST support delete.
```

Comment:
Since you've splitted this rule into two rules, could you split this doc into two as well ?